cats/CI_api_query.py

```diff
@@ -1,13 +1,9 @@
 import requests_cache
-from typing import Callable
 from datetime import datetime, timezone
 
+from .CI_api_interface import API_interfaces
 
-def get_tuple(
-    postcode: str,
-    request_url: Callable[[datetime, str], str],
-    parse_data_from_json: Callable[[dict], list[tuple[datetime, int]]],
-) -> list[list[tuple[datetime, int]]]:
+def get_CI_forecast(postcode: str) -> list[list[tuple[datetime, int]]]:
```

Comment:
Should return a `list[CarbonIntensityPointEstimate]`

Reply:
Changed in 4bf61e9fa115de213849855c037e199da155d43e

Reply:
Changed in 4bf61e9fa115de213849855c037e199da155d43e